src/events/guildMemberAdd.ts

```diff
@@ -18,9 +18,10 @@ export default TypedEvent({
             .setColor("ORANGE")
             .setTitle("New Member")
             .setDescription(
-                `Welcome ${member.user.username} to the conaticus server\n` +
-                    "Use `/rolemenu` to choose your pings and languages roles\n" +
-                    "Enjoy your stay!"
+                `Welcome ${member.user.username} to Conaticus' discord server\n` +
```

Comment:
This bot is general-purpose rather than specialized. To fix this we can replace `Conaticus' discord server` with the guild name.

Reply:
Would it be better if I created a new command which allows for customising the welcome message on a server-to-server basis?

Reply:
nobody wants to take the time to do that

Reply:
Yes, but there must be a default response if the server hasn't set it yet.